frontend/src/redux/types.ts

```diff
@@ -11,7 +11,6 @@ export enum Actions {
   FORCE_COMPONENTS_UPDATE = 'FORCE_COMPONENTS_UPDATE',
 }
 
-export type AppNotificationStatus = 'success' | 'danger' | 'warning' | 'info' | 'default';
```

Comment:
Umm... maybe just undo all of this... not sure what you were changing 🤔 I think these values are fine.

Reply:
Updated to reference the PF [`AlertVariant` enum](https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/components/Alert/Alert.tsx#L12) let me know if this is what you had in mind @andrewballantyne